pinot-core/src/test/resources/data/test_presence_vector_data.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
should we also test the case something like

`"description" : null`?

Reply:
I wanted to leave one column non-null (in this case description).